infra/auth-server/test/index.js

```diff
@@ -0,0 +1,249 @@
+'use strict'
+
+const chai = require('chai')
+const expect = chai.expect
+const request = require('supertest')
+
+const crypto = require('crypto')
+
+const { generateToken, verifyToken } = require('@origin/auth-utils/src/utils')
+
+const app = require('../src/app')
+
+const Eth = require('web3-eth')
+
+const stringify = require('json-stable-stringify')
+
+const Web3Eth = new Eth()
```

Comment:
Again, could this just be `const Web3Eth = require('web3-eth')` ?

Reply:
Correct me if I'm wrong. But you could only use `web3-utils` without creating a new instance. You have to create an instance for `web3-eth`.  [Ref.](https://web3js.readthedocs.io/en/v1.2.2/)